compiler/rustc_codegen_cranelift/src/debuginfo/mod.rs

```diff
@@ -77,9 +77,16 @@ impl DebugContext {
                 FileNameDisplayPreference::Local
             })
             .into_owned();
```

Comment:
Since `working_dir` is a  `RealFileName` we can do:
```rust
let comp_dir = tcx
    .sess
    .opts
    .working_dir
    .for_scope(tcx.sess, RemapPathScopeComponents::DEBUGINFO)
    .to_string_lossy();
```